static/js/analytics.js

```diff
@@ -153,6 +153,7 @@ function analyticsSurveyLogic() {
   switch (surveyType) {
     case "nps": {
       const notLikely = document.createElement("li");
+      // TODO: Translate these strings
```

Comment:
We need to add Fluent  support to the JavaScript for this function.

Reply:
FWIW, my approach in Monitor for translating the JS-powered micro-survey responses was to add data attributes for the strings (e.g., `data-{{ msg_id }}-translated="{% ftlmsg msg_id %}"`) and then have the JS put that value into the `textContent` of the proper element.